src/main/java/com/misternerd/resteasytox/javascript/ResteasyToJavascriptConverter.java

```diff
@@ -331,8 +352,14 @@ private void createHttpMethodCall(ServiceMethod serviceMethod, JavascriptPublicM
 		// GET requires no body
 		if(GET == serviceMethod.httpMethod)
 		{
-			method.addBody("var request = restClient.getRequest(PATH + '%s', headerParams, pathParams, '%s', '%s', %s);",
-				serviceMethod.path, serviceMethod.requestContentType, serviceMethod.responseContentType, returnType);
+			String urlStr;
+			if(!serviceMethod.queryParams.isEmpty()) {
```

Comment:
@SergeKorobko I think it makes sense to support query params for all types of request. Please move this URL generating block up and use it for all method types.